### ISSUE TITLE
can't list open tasks

### ISSUE THREAD

**Issue by ecocode** [chaosteil/doist#24]
I just installed doist from cargo and authenticated with the API key successfully.  Adding tasks works, but listing them doesn't:

```
 ~ ❯ RUST_BACKTRACE=full doist l -n
Error:
   0: unable to get labels
   1: Unable to parse API response
   2: invalid value: 7, expected one of: 0, 30, 31, 32, 33, 34, 35, 36, 37, 38, 39, 40, 41, 42, 43, 44, 45, 46, 47, 48, 49 at line 6 column 12

Location:
   /var/home/ec/.cargo/registry/src/github.com-1ecc6299db9ec823/doist-0.1.12/src/api/rest/gateway.rs:227

  ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ BACKTRACE ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━
                                ⋮ 3 frames hidden ⋮
   4: <core::future::from_generator::GenFuture<T> as core::future::future::Future>::poll::hfc058abe79228d17
      at <unknown source file>:<unknown line>
   5: <tokio::future::maybe_done::MaybeDone<Fut> as core::future::future::Future>::poll::he322455f7880ab63
      at <unknown source file>:<unknown line>
   6: <tokio::future::poll_fn::PollFn<F> as core::future::future::Future>::poll::h97685c0f2aa06a41
      at <unknown source file>:<unknown line>
   7: <core::future::from_generator::GenFuture<T> as core::future::future::Future>::poll::hedd34811e8c38de6
      at <unknown source file>:<unknown line>
   8: <core::future::from_generator::GenFuture<T> as core::future::future::Future>::poll::h7fed7fe687d641d1
      at <unknown source file>:<unknown line>
   9: std::thread::local::LocalKey<T>::with::hf21868b65357e4b2
      at <unknown source file>:<unknown line>
  10: tokio::park::thread::CachedParkThread::block_on::h4f86d4b78618c24e
      at <unknown source file>:<unknown line>
  11: tokio::runtime::thread_pool::ThreadPool::block_on::h5a0e2a74867af865
      at <unknown source file>:<unknown line>
  12: tokio::runtime::Runtime::block_on::h0541b814246c355f
      at <unknown source file>:<unknown line>
  13: doist::main::hbed4e83e0c3cac41
      at <unknown source file>:<unknown line>
  14: std::sys_common::backtrace::__rust_begin_short_backtrace::h37e1a665d7d0673d
      at <unknown source file>:<unknown line>
  15: std::rt::lang_start::{{closure}}::he92c7341e2eb2d06
      at <unknown source file>:<unknown line>
  16: std::rt::lang_start_internal::hf811e02ab96793d6
      at <unknown source file>:<unknown line>
  17: main<unknown>
      at <unknown source file>:<unknown line>
  18: __libc_start_call_main<unknown>
      at <unknown source file>:<unknown line>
  19: __libc_start_main_alias_2<unknown>
      at <unknown source file>:<unknown line>
  20: _start<unknown>
      at <unknown source file>:<unknown line>
```

**Comment by chaosteil** [comment]
Ah, that's wild, your labels have colors that are not documented! Let me see how I can unblock you real quick.

**Comment by chaosteil** [comment]
And just published 0.1.13 on crates.io, hope that fixes your issue!

**Comment by ecocode** [comment]
the issue is solved but got another one ;)  I will put this in another issue
